packages/bloc/pubspec.yaml

```
@@ -1,20 +1,25 @@
 name: bloc
 description: A predictable state management library that helps implement the BLoC (Business Logic Component) design pattern.
-version: 6.0.3
+version: 6.1.0-nullsafety
 repository: https://github.com/felangel/bloc/tree/master/packages/bloc
 issue_tracker: https://github.com/felangel/bloc/issues
 homepage: https://github.com/felangel/bloc
 documentation: https://bloclibrary.dev
 
+# Can't be published until null safety is shipped without requiring a flag
+publish_to: none
+
 environment:
-  sdk: ">=2.6.0 <3.0.0"
+  sdk: ">=2.10.0-99.0.dev <2.10.0"
```

```suggestion
  sdk: ">=2.10.0-99.0.dev <2.11.0"
```

We now have the 2.11 dev sdk which you can safely allow.